test/test-project/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+#required by GitHub Actions
+sudo chown -R 1001:121 "/root/.npm"
+cd app
+npm install
+cd test/test-project
+npm install
+npm run test:project
+#installing html template globally before global tests
+PUPPETEER_SKIP_DOWNLOAD=true npm install -g @asyncapi/html-template@0.16.0
+#remove previously installed html template to make sure it is not picked up in the test
+rm -rf node_modules/@asyncapi/html-template
```

Comment:
```suggestion
rm -rf ./node_modules/@asyncapi/html-template
```

As I remember some commands (I don't remember if `rm` also) treat path without `./` prefix as global path.

Reply:
@magicmatatjahu you want me to check it or you already did and this is why you made a code change suggestion 😃

Reply:
so I guess I will not change it if it works this way? 😛